ddl/utils.lua

```diff
@@ -155,5 +162,6 @@ return {
     redundant_key = redundant_key,
     find_first_duplicate = find_first_duplicate,
     lj_char_isident = lj_char_isident,
+    lj_char_isdigit = lj_char_isdigit,
```

Comment:
Don't you consider using isdigit function from utf8 module? (https://www.tarantool.io/ru/doc/latest/reference/reference_lua/utf8/#utf8-isdigit)

Reply:
No-no, let's do it consistently, please. I don't want one symbol check function from one source/lib and another from another one.